ds3-cli-integration/src/main/resources/META-INF/services/com.spectralogic.ds3cli.command.CliCommand

```diff
@@ -0,0 +1,41 @@
+# CLI Command implementatiosn in this jar
```

Comment:
Do you need this in the integration jar file?  Shouldn't the ServiceProvider look this up in the other jar file?